_layouts/default.html

```diff
@@ -1,38 +1,66 @@
-    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
-    <![endif]-->
   </head>
+
   <body>
-      {{ content }}
-  <p>Autor: <a href="https://www.linkedin.com/in/lucie-karmova/">Lucie Karmova</a></p>
-  <p><small>Hosted on GitHub Pages &mdash; Theme by <a href="https://github.com/orderedlist">orderedlist</a></small></p>
-  <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/4.0/">
-      <img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-nd/4.0/88x31.png" />
-  </a><br/>
-This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License</a>.
-    </footer>
-    <script src="{{ '/assets/js/scale.fix.js' | relative_url }}"></script>
-  {% if site.google_analytics %}
-    <script>
+
+    <!-- HEADER -->
+    <div id="header_wrap" class="outer">
+        <header class="inner">
+        <!--  <a id="forkme_banner" href="{{ site.github.repository_url }}">View on GitHub</a> -->
+
+          <h1 id="project_title">{{ site.title | default: site.github.repository_name }}</h1>
+
+      <h4 id="project_tagline"><em>pro nemocnice, sociální pracovníky a ošetřovatele</em></h4> 
+			<a  href = "https://en.sijemerousky.cz" id = "head_anchor">Anglická verze</a>
+		<a  href = "mailto:sijemerousky@gmail.com" id = "contact_anchor">sijemerousky@gmail.com</a>
+          {% if site.show_downloads %}
+            <section id="downloads">
+              <a class="zip_download_link" href="{{ site.github.zip_url }}">Download this project as a .zip file</a>
+              <a class="tar_download_link" href="{{ site.github.tar_url }}">Download this project as a tar.gz file</a>
+            </section>
+          {% endif %}
+        </header>
+    </div>
+
+    <!-- MAIN CONTENT -->
+    <div id="main_content_wrap" class="outer">
+      <section id="main_content" class="inner">
+        {{ content }}
+      </section>
+    </div>
+
+    <!-- FOOTER  -->
+    <div id="footer_wrap" class="outer">
+      <footer class="inner">
+        {% if site.github.is_project_page %}
+        <p class="copyright">{{ site.title | default: site.github.repository_name }} maintained by <a href="{{ site.github.owner_url }}">{{ site.github.owner_name }}</a></p>
+        {% endif %}
+        <p>Published with <a href="https://pages.github.com">GitHub Pages</a></p>
+      </footer>
+    </div>
+
+    {% if site.google_analytics %}
+      <script>
         (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
         m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
+        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
```

Comment:
@BecAlenvers 
Is this really correct?  
```
//www
```
instead of
```
https://
```

I believe this is the third time we are iterating on this tiny bit.

Reply:
True that